### DISCUSSION TITLE
v0.80.0

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#7632] [Announcements]
### Improvements

* Added a workspace preference for admins to disable individual user account removal
* New Umami analytics integration by @QYG2297248353 in https://github.com/outline/outline/pull/7366
* It is now possible to @mention users that do not have access to a document (they won't be notified) in https://github.com/outline/outline/pull/7601
* Behavior of `Tab` and `Shift-Tab` is now more natural in code blocks https://github.com/outline/outline/pull/7622
* "Public access" section of share dialog is now always visible in https://github.com/outline/outline/pull/7617
* Archiving a document now includes a confirmation dialog
* `CMD-k` with the cursor within an editor link now focuses the link field
* Home, Search, and Drafts are now sticky in the sidebar
* `CMD-Shift-c` is now a shortcut for inline code

### Fixes

* Individual tabs will be restored to the correct location when re-logging in
* Reduced data loading requirements in https://github.com/outline/outline/pull/7442
* Improved results for user names with diacritics in @mention search in https://github.com/outline/outline/pull/7590
* Finding assignee in search filters now works correctly when workspace has more than 100 users in https://github.com/outline/outline/pull/7597
* Fixed revision history being lost after deleting a document and restoring it
* Fixed an issue in latest Chrome when typing Korean characters https://github.com/outline/outline/issues/7574
* Fixed OIDC signin to prevent duplicate auth providers in https://github.com/outline/outline/pull/7598
* It is now possible to drag and drop into "Shared with me" in https://github.com/outline/outline/pull/7619
* Fixed `documents.archived` no longer errors with `sort: "index"` in request in https://github.com/outline/outline/pull/7628
* Fixed `document.content` column not updated when sending text attribute through API in https://github.com/outline/outline/pull/7630
* Fixed an issue where document editors are sometimes not included in insights and `collaboratorIds`
* Fixed archived documents appearing in "Shared with me" sidebar section
* Fixed collection description cannot be completely removed
* Added resilience for absolute attachment paths in Markdown importer
* Drafts in sidebar will now show "+" when there are more than 25 drafts
* Improved loading behavior of empty collections
* Fixed an issue with link opening when linking to a website on the same host with a different port
* Fixed display of negative days in deletion notice banner

## New Contributors
* @QYG2297248353 made their first contribution in https://github.com/outline/outline/pull/7366

**Full Changelog**: https://github.com/outline/outline/compare/v0.79.1...v0.80.0

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.80.0'>v0.80.0</a>.</em>

**Comment by sumadark** [comment]
Hello @tommoor ,

Thank you once again for this amazing tool I am using every day for almost 3 years now ❤️
Just updated to v0.80.0 from v.0.79.1, image was correctly pushed, but in the preferences I can still see that I am using v0.79.1 

Is there something specific I should do to handle this new release ? As I have never experienced this situation before. 
I am using Outline with Docker compose file, no change except the version number of the image to push.

Thank you.

Suma

**Reply by tommoor** [reply]
The version is incorrect inside the docker image, my bad nothing to worry about though.
